content/about/faq.md

```diff
@@ -1,23 +1,54 @@
 ---
-title: FAQ
+title: FAQs
```

Comment:
Maybe need a little preamble/abstract at the beginning to clarify that these are specifically FAQs about our move to 11ty?